app/com/lucidchart/open/cashy/utils/DownloadHelper.scala

```diff
@@ -6,17 +6,22 @@ import org.apache.http.impl.client.HttpClientBuilder
 
 case class DownloadFailedException(message: String) extends Exception(message)
 
+case class DownloadResult(
+  bytes: Array[Byte],
+  contentType: Option[String]
+)
+
 object DownloadHelper extends DownloadHelper
 class DownloadHelper {
 
-  def downloadBytes(url: String): Array[Byte] = {
+  def download(url: String): DownloadResult = {
     val httpClient = HttpClientBuilder.create().build()
 
     val httpGet = new HttpGet(url)
     val response = httpClient.execute(httpGet)
 
     val statusCode = response.getStatusLine().getStatusCode()
-    if (statusCode != 200) {
+    val bytes = if (statusCode != 200) {
```

Comment:
I would prefer to leave the if as it was. Assigning `val bytes` to the value of this if statement is misleading cause it makes it seem as if we're getting a value from either branch of the conditional.